atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -47,15 +49,13 @@ public File acceptorLogDir() {
     public abstract String localServer();
 
     @Size(min = 1)
-    public abstract Set<String> leaders();
+    public abstract List<String> leaders();
```

Comment:
I agree with Schlosnagle here.  Is there a compelling reason why the list route is better?